### DISCUSSION TITLE
Document default keybindings on the website

### DISCUSSION THREAD

**Discussion by emosenkis** [zellij-org/zellij#1973] [Ideas]
Currently, I have to dump the config to find out the keybindings

**Comment by sommerper** [comment]
I was just searching for the same thing. Would be pretty handy to have.

**Comment by sommerper** [comment]
But now that I had a look I still can't find the keybinding I'm looking for.
Does anyone know what command is being executed when pressing ctrl-del? It switches two horizontally divided to vertically divided. I can't find that documented anywhere.
